scripts/sanity_chk/sanitylib.py

```diff
@@ -336,6 +336,7 @@ def __init__(self, instance, type_str):
         super().__init__(instance, type_str)
 
         self.terminated = False
+        self.call_west_flash = False
```

Comment:
AFAIK, `west flash` for the purpose of running a binary on the host works only with nsim, for example we can't call `west flash` on qemu or renode.

Reply:
I was under impression that we can call west flash on qemu as we have `scripts/west_commands/runners/qemu.py`. But I've checked it and it appears that it is only a stub.

I am wondering why do we have it?

Reply:
For ARC, this has historical reasons. As we didn't support flashing of our development boards (EMSK e.g. doesn't have flash), the 'flash' target was implemented by downloading and running the executable instead of flash&run. For nSIM this then was copied.

Should a 'west run' be introduced to be more consistent, instead of the 'make run'?
Using mdb in 'make run' isn't straightforward, as I understood from Evgeniy.

Reply:
> I am wondering why do we have it?

Various people keep meaning to make it not a stub :). You can remove it if you really want but I don't think it does any harm. It would be nice to have it, just nobody has had time to do it.